adapters/tlsio_mbedtls.c

```diff
@@ -511,7 +543,7 @@ CONCRETE_IO_HANDLE tlsio_mbedtls_create(void *io_create_parameters)
                 {
                     result->tls_status = TLS_STATE_NOT_INITIALIZED;
                     mbedtls_init((void*)result);
-
+                    memset((void*)&result->send_complete_info, 0, sizeof(result->send_complete_info));
```

Comment:
this seems to be doing duplicate work that `calloc` is doing

Reply:
will remove.

---
In reply to: [354466164](https://github.com/Azure/azure-c-shared-utility/pull/421#discussion_r354466164) [](ancestors = 354466164)